src/components/Contentful/SectionWithBackgroundClassic.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+	<section class="">
+		<div class="tw-relative tw-p-0">
+			<div
+				class="tw-relative tw-w-full tw-overflow-hidden tw-z-1 tw-m-0 tw-top-0"
+				:class="verticalPaddingClasses"
+			>
+				<slot name="content">
+				</slot>
+			</div>
+			<div
+				class="tw-w-full tw-h-full tw-absolute tw-top-0 tw-z--1"
+				:style="backgroundStyle"
+			>
+				<video class="tw-w-full tw-h-full tw-object-cover" v-if="isBackgroundVideo"
+					:src="backgroundMedia.url"
+					autoplay
+					loop
+					muted
+					playsinline
+				></video>
+				<kv-contentful-img class="tw-w-full tw-h-full tw-object-cover"
+					v-if="isBackgroundImage"
+					:width="1440"
+					:contentful-src="backgroundMedia.url"
+					fallback-format="jpg"
+					:alt="backgroundMedia.description"
+					:source-sizes="sourceSizes"
+				/>
+			</div>
+		</div>
+	</section>
+</template>
+
+<script>
+import KvContentfulImg from '~/@kiva/kv-components/vue/KvContentfulImg';
```

Comment:
Updated to use kv-components version which now has some additions to fix the image cover styles once that  repo is released and updated here.

Reply:
The release happens on each merge in kv-ui-elements, so should be ready a package.json bump in this PR

Reply:
This feature isn't used on Homepage so it's ok to come in later with the update kv-ui-elements resulting from the new carousel work.